cpg_utils/config.py

```diff
@@ -4,29 +4,29 @@
 from typing import Optional, List, Dict
 
 import toml
-from cloudpathlib import AnyPath
 from frozendict import frozendict
 
+from cpg_utils import to_path
 
 # We use these globals for lazy initialization, but pylint doesn't like that.
 # pylint: disable=global-statement, invalid-name
 _config_paths = _val.split(',') if (_val := os.getenv('CPG_CONFIG_PATH')) else []
 _config: Optional[frozendict] = None  # Cached config, initialized lazily.
 
 
-def _validate_configs(config_paths: List[str]) -> None:
+def _validate_configs(config_paths: list[str]) -> None:
```

Comment:
I know this isn't the right PR for this comment, but I'd like this validation method to do some more validation 😬 

Any thoughts on whether we should attempt to load each identified file using `toml` to check that they both exist and can be parsed correctly?

Reply:
Yeah, good idea. Loading and parsing from remote might slow down the application a bit, but is probably worth it

Reply:
I added some validation into that function